recipes/karney/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "karney" %}
+{% set version = "1.1.0" %}  # This should be the version you intend to release
+{% set python_min = "3.9" %}
```

Comment:
You shouldn't set `python_min` when it is the same as the global conda-forge pin of "3.9"

Reply:
@synapticarbors I have removed the "{% set python_min = "3.9" %}" line.